packages/edit-site/src/components/page-pages/index.js

```diff
@@ -31,6 +32,12 @@ export default function PagePages() {
 			field: 'date',
 			direction: 'desc',
 		},
+		fields: {
+			// All fields are visible by default, so it's
+			// better to keep track of the hidden ones.
+			hidden: new Set( [ 'date' ] ),
+			hideable: [ 'author', 'status', 'date' ],
```

Comment:
I'm not yet sure if we should keep track of fields that aren't hidable. I think future work will inform us about it..

Reply:
commented about this here https://github.com/WordPress/gutenberg/pull/55247#discussion_r1354590037

I think this is similar to "enableSorting" and should be part of the fields config instead of the view config.

Reply:
Makes sense, yes. I'll update.